R/post.R

```diff
@@ -65,6 +65,7 @@
 #' }
 #' @family post
 #' @aliases post_status
+#' @importFrom lifecycle deprecate_warn
```

Comment:
You don't need to import this since you're using it with `::`

Reply:
Does it have any downsides?

Reply:
No downsides — you should only either use `::` or `@importFrom`, not both.